### ISSUE TITLE
it's hard to automate alterting based on "info" and "quorum" endpoints

### ISSUE THREAD

**Issue by MonsieurNicolas** [stellar/stellar-core#2187]
Both `quorum` and `info` report the latest information gathered by SCP.

The problem is that when calling those endpoints at the beginning of an SCP round, most of that information is misleading:
For example most validators would show up as "missing" as the node didn't have time yet to receive and process any message for the new slot.

To work around this, in logs we already give information on the round (2 ledgers ago):
```
2019-07-09T19:16:10.853 GCM6Q [Herder INFO] Quorum information for 24750505 : {"agree":17,"delayed":0,"disagree":0,"fail_at":4,"hash":"b69f17","ledger":24750505,"missing":0,"phase":"EXTERNALIZE","validated":true}

2019-07-09T19:16:10.862 GCM6Q [Ledger INFO] Got consensus: [seq=24750507, prev=3a820b, txs=19, ops=190, sv: [ txH: e2f514, ct: 1562699769, upgrades: [ ] ]]
```

I am not sure what the best strategy is for those endpoints:
* probably have an option to report the latest version
* change the default to do something a little smarter, like if the current SCP round was started less than 3(?) seconds ago, report information on the previous slot, otherwise report information on the current slot

**Comment by andrenarchy** [comment]
I think always reporting the last closed ledger makes sense.

**Comment by MonsieurNicolas** [comment]
that's a possibility @andrenarchy , the problem with only reporting things from the previous ledger is that it means we're blind on the currently in flight ledger (where issues are most critical). So if things go sideways (and the ledger doesn't close) you won't know why because what is reported is from now stale data. I guess we'd see it with the node not being "in sync" anymore, so to get diagnostics we could call the `quorum` endpoint that would return the latest data

**Comment by andrenarchy** [comment]
Yes, I think you'd check `quorum` then. Another idea would be to offer both: one `current` ledger and a `last` ledger entry so you can pick what you need.
